pandas/tests/io/sas/test_xport.py

```diff
@@ -127,7 +131,8 @@ def test2_binary(self):
         data_csv = pd.read_csv(self.file02.replace(".xpt", ".csv"))
         numeric_as_float(data_csv)
 
-        data = read_sas(self.file02b, format="xport")
+        with open(self.file02, "rb") as fd:
+            data = read_sas(fd, format="xport")
```

Comment:
why is this actually leaking?

Reply:
IIUC ATM we're passing an open file and the read_sas call closes it incorrectly, so its an anti-leak

Reply:
this doesn't make sense; there is a bug in the xport reader; paper it over here is not a great soln.